packages/data/src/lib/data-view/data-view.spec.ts

```diff
@@ -0,0 +1,173 @@
+import { CartoError } from '@carto/toolkit-core';
+import { Layer } from '@carto/toolkit-viz';
+import { DataView } from './data-view';
+import { AggregationTypes } from '../aggregations/aggregations';
+
+describe('DataView', () => {
```

Comment:
We should probably create a tests folder like in other packages...

Reply:
No, I think we should leave tests right next to their source files to make them easier to find. I didn't like the split folder for tests, and tried to change it in the visualization module but I couldn't and now tried again and succeed. So I think we should do the same with all modules.

Reply:
🆗 I also like the idea of having them side by side, but then let's add a small task to make the same in other packages

Reply:
But we already have in a separate folder... so we will have 2 ways. I would add tests in the __test__ folder by now and then, if we want to change them, do it in a different PR